src/wasm/wasm-s-parser.cpp

```diff
@@ -948,6 +976,11 @@ void SExpressionWasmBuilder::preParseHeapTypes(Element& module) {
       Element& subtypeKind = *subtype[0];
       if (subtypeKind == FUNC) {
         builder[index] = parseSignatureDef(subtype, 0);
+      } else if (kind == CONT) {
+        // FIXME(frank-emrich) Should we support this?
+        throw ParseException("continuation type in subtype definition",
+                             subtypeKind.line,
+                             subtypeKind.col);
```

Comment:
For now I think it is fine to error. We want to have the proper design in place in the reference interpreter with the GC extension before we start implementing anything in the toolchains.